test/functional/feature_proxy.py

```diff
@@ -106,7 +109,7 @@ def setup_nodes(self):
             ['-listen', f'-proxy={self.conf2.addr[0]}:{self.conf2.addr[1]}','-proxyrandomize=1'],
             [],
             ['-listen', f'-proxy={self.conf1.addr[0]}:{self.conf1.addr[1]}','-proxyrandomize=1',
-                '-cjdnsreachable']
+                '-cjdnsreachable', '-yggdrasilreachable']
```

Comment:
0a33d6c only had a quick look, so to be confirmed, but it looks like you need to do something like this for the functional test to pass
```diff
 class ProxyTest(BitcoinTestFramework):
     def set_test_params(self):
-        self.num_nodes = 5
+        self.num_nodes = 6
         self.setup_clean_chain = True
 
     def setup_nodes(self):
@@ -109,7 +109,9 @@ class ProxyTest(BitcoinTestFramework):
             ['-listen', f'-proxy={self.conf2.addr[0]}:{self.conf2.addr[1]}','-proxyrandomize=1'],
             [],
             ['-listen', f'-proxy={self.conf1.addr[0]}:{self.conf1.addr[1]}','-proxyrandomize=1',
-                '-cjdnsreachable', '-yggdrasilreachable']
+                '-cjdnsreachable'],
+            ['-listen', f'-proxy={self.conf1.addr[0]}:{self.conf1.addr[1]}','-proxyrandomize=1',
+                '-yggdrasilreachable'],
         ]
```

Reply:
Addressed in 578ffd92b774f33391d77b9c00366a30b27d9611